test/csv/test_table.rb

```diff
@@ -336,6 +336,78 @@ def test_to_csv
     assert_equal(csv, @header_table.to_csv)
   end
 
+  def test_to_csv_with_limit_set_to_less_than_the_number_of_rows
```

Comment:
How about simplifying test names?

* `test_to_csv_limit_positive`
* `test_to_csv_limit_positive_over`
* `test_to_csv_limit_zero`
* `test_to_csv_limit_negative`
* `test_to_csv_limit_negative_over`

Full English described test names will be easy to read for fluent English speakers but they aren't easy to read for me (a non fluent English speaker). Because I need to read English carefully...

Simple different only test names are easy to read for me because I need to read less English words. I hope that fluent English speakers also can be easy to read...

Reply:
Sure @kou I'll update